packages/dev-tools/src/styles/dev-tools.css

```
@@ -0,0 +1,347 @@
+	max-height: 16rem;
+	margin: 0;
+	overflow: auto;
+	padding: 0.75rem;
+	border-radius: 0.5rem;
+	background: var(--c15t-dev-tools-bg-muted);
+	color: var(--c15t-dev-tools-text);
+	font-family: ui-monospace, SFMono-Regular, Menlo, Consolas, monospace;
+	font-size: 0.75rem;
+	line-height: 1.5;
+	white-space: pre-wrap;
+	word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)